modules/502-delivery/docs/USAGE_RU.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "Модуль delivery: пример конфигурации"
+---
+
+- [Дисклеймер](#дисклеймер)
+- [Концепция](#концепция)
+- [Конфигурация](#конфигурация)
+  - [WerfSource CRD](#werfsource-crd)
+  - [Частичное использование WerfSource CRD](#частичное-использование-werfsource-crd)
+    - [Без репозитория Argo CD](#без-репозитория-argo-cd)
+    - [Без регистри для Image Updater](#без-регистри-для-image-updater)
+- [Публикация артефакта в registry](#публикация-артефакта-в-registry)
+- [Деплой](#деплой)
+- [Автообновление бандла](#автообновление-бандла)
+  - [Правила обновлений образов](#правила-обновлений-образов)
+  - [Настройки доступа к registry](#настройки-доступа-к-registry)
+    - [Индивидуальная настройка для Application](#индивидуальная-настройка-для-application)
+- [Особенности аутентификации CLI](#особенности-аутентификации-cli)
+- [Как создать OCI-репозиторий самостоятельно](#как-создать-oci-репозиторий-самостоятельно)
+  - [Веб-интерфейс и kubectl](#веб-интерфейс-и-kubectl)
+  - [Командная утилита `argocd`](#командная-утилита-argocd)
+
+## Дисклеймер
+
+Мы ожидаем, что читатель знаком с Argo CD, поэтому целью этой статьи является описание особенностей работы в поставке с Deckhouse.
```

Comment:
```suggestion
Мы ожидаем, что читатель знаком с Argo CD, поэтому целью этой статьи является описание особенностей работы в поставке с Deckhouse.
```
This is not an article but a documentation page.

Reply:
I dont't think it is necessarily a mismatch. Page is the medium format, article is the content format, documentation is the purpose. I know that I'm reading documentation article on a webpage. Anyway, I'll accept any other terminology that is consistent with the existing content.